tc/core/polyhedral/codegen_llvm.cc

```diff
@@ -302,15 +311,54 @@ class CodeGen_TC : public Halide::Internal::CodeGen_X86 {
   }
 };
 
-llvm::Value* CodeGen_TC::getValue(isl::ast_expr expr) {
+Halide::Expr CodeGen_TC::makeHalideExpr(isl::ast_expr expr) {
   if (auto idExpr = expr.as<isl::ast_expr_id>()) {
-    return sym_get(idExpr.get_id().get_name());
+    return Halide::Internal::Variable::make(
+        Halide::Int(32), idExpr.get_id().get_name());
```

Comment:
For now, it may be safe to assume isl AST ids correspond to `int32_t` variables, but it may not necessarily be the case in the future.

Reply:
I'll add a comment, we may need 64b earlier than we think